tasks/copy_static_files.yml

```
@@ -0,0 +1,18 @@
+---
+- name: Copy static files
```

```suggestion
- name: Copy subdomain static files
```

Also can this be renamed to something like `tasks/copy_subdomain_static_files.yml`?